src/servers/main.cc

```diff
@@ -366,6 +368,27 @@ std::vector<Option> options_
        "Name of the model to be loaded on server startup. It may be specified "
        "multiple times to add multiple models. Note that this option will only "
        "take affect if --model-control-mode=explicit is true."},
+      {OPTION_RATE_LIMIT, "rate-limit", Option::ArgStr,
+       "Specify the mode for rate limiting. Options are \"execution_count\" "
```

Comment:
I think the default should be "off"

Reply:
I might have confused something. 
Can you explain what did you meant here: 
https://github.com/triton-inference-server/core/pull/6#discussion_r515357510

Reply:
You are right, I got confused. It should be on by default... so leave this like you have it.